app/auto-updater-linux.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const fetch = require('electron-fetch');
+const fetch = require('electron-fetch').default;
```

Comment:
This is due to `electron-fetch` version bump right?

Reply:
Yes. I tried using `import` but it didn't work so I'm assuming this is not being transpiled